src/components/Router/index.js

```diff
@@ -137,7 +137,7 @@ class CrossDeviceMobileRouter extends Component {
 
   sendClientSuccess = () => {
     this.state.socket.off('custom disconnect', this.onDisconnect)
-    const { faceCapture } = this.props
+    const { captures: { face: faceCapture } } = this.props
```

Comment:
On cross device client success we need to send the document id back to the desktop so it can returned in the onComplete callback

Reply:
Hmm, I'm not sure I 100% understand your comment?
I shouldn't have changed any existing data being transferred - just added some extra data
Can we chat offline?

Reply:
Discussed this offline - this `sendClientSuccess` method is used in the cross-device flow, where we currently just send the facial information across to the main window.

We need to, as a part of this PR, also send the document information across, for the `onComplete` hook to benefit from.

I've added a new commit to handle this. Cheers for pointing it out!